grafana/grafana
# postgresql: change plugin id

(part of https://github.com/grafana/grafana/issues/72892)

to move the core postgres plugin closer to external plugins, it's plugin id ( `postgres` ) has to change, and become `grafana-*-datasource`. we set the old name (`postgres`) as an alias, so things should work.

scenarios to test:
- create dashboard before-change, open it after-change, it should work
- create alert before-change, open it after-change, it should work
- create provisioned dashboard with old-plugin-id, load after-change, it should work
